### DISCUSSION TITLE
Export and import Settings

### DISCUSSION THREAD

**Discussion by Dnarts** [sezanzeb/input-remapper#852] [Ideas]
Hey everyone, thanks for your wonderful service. It's been amazing so far.

I've been using it for about half a year now. I'm a bit concerned about potential issues with my Ubuntu system, since I'm new to it. I'm worried about losing my settings and having to start from scratch, which would be a huge hassle, especially with all the devices I have connected (gamepads, keyboards, trackball, etc.).

It would be great if there was a way to back up my settings. Thanks in advance!

**Comment by lejp** [comment]
I'm also interested. Is there any config file where all mapped keys are stored. So it can be simply copied to the new system.
